app/hooks/contextHooks.ts

```diff
@@ -0,0 +1,13 @@
+import { useOutletContext } from '@remix-run/react';
+import { AppContext } from '~/typer/context';
+import { ESanitySteg } from '~/typer/sanity/sanity';
+
+export function useFornavn() {
+  const { brukerdata } = useOutletContext<AppContext>();
+  return brukerdata.fornavn;
+}
```

Comment:
Jeg synes denne funksjonen er litt overkill. Dersom vi ette hvert får mange felter på brukeren blir det en del jobb om vi skal ha en en egen funksjon per felt i brukerdataene. 

For min del tenker jeg det er vel så enkelt å bare skrive `const { brukerdata } = useOutletContext<AppContext>();` i `VeilederHilsen` og så bruker `brukerdata.fornavn` der vi trenger fornavnet

Reply:
Santsant, ble litt endret oppsett når jeg endret den andre kommentaren også [3ad4a88](https://github.com/bekk/nav-familie-endringsmelding/pull/20/commits/3ad4a88e8f95e23b4a14c0a3459e30a6c994cc16)